compair/tests/algorithms/test_pair_adaptive_min_delta.py

```diff
@@ -0,0 +1,462 @@
+        min_key = min([results.key1, results.key2])
+        max_key = max([results.key1, results.key2])
+
+        self.assertEqual(min_key, 1)
+        self.assertEqual(max_key, 2)
+        self.assertEqual(results.winner, None)
+
+        ##
+        # Selects lowest round objects first
+        scored_objects = [
+            ScoredObject(
+                key=1, score=0.5, variable1=None, variable2=None,
+                rounds=2, wins=None, loses=None, opponents=None
+            ),
+            ScoredObject(
+                key=2, score=0.5, variable1=None, variable2=None,
+                rounds=2, wins=None, loses=None, opponents=None
+            ),
+            ScoredObject(
+                key=3, score=0.5, variable1=None, variable2=None,
+                rounds=2, wins=None, loses=None, opponents=None
+            ),
+            ScoredObject(
+                key=4, score=0.5, variable1=None, variable2=None,
+                rounds=2, wins=None, loses=None, opponents=None
+            ),
+            ScoredObject(
+                key=5, score=0.5, variable1=None, variable2=None,
+                rounds=1, wins=None, loses=None, opponents=None
+            ),
+            ScoredObject(
+                key=6, score=0.5, variable1=None, variable2=None,
+                rounds=1, wins=None, loses=None, opponents=None
+            )
+        ]
+        comparisons = []
+        criterion_scores = {
+            1: { 'c1': 100, 'c2': 200 },
+            2: { 'c1': 100, 'c2': 200 },
+            3: { 'c1': 100, 'c2': 200 },
+            4: { 'c1': 100, 'c2': 200 },
+            5: { 'c1': 100, 'c2': 200 },
+            6: { 'c1': 100, 'c2': 200 }
+        }
+        criterion_weights = {
+            'c1': 0.7, 'c2': 0.3
+        }
+
+        results = self.pair_algorithm.generate_pair(scored_objects, comparisons,
+            criterion_scores, criterion_weights)
+
+        # 5 & 6 should be selected as the lowest valid round objects
+        min_key = min([results.key1, results.key2])
+        max_key = max([results.key1, results.key2])
+        self.assertEqual(min_key, 5)
+        self.assertEqual(max_key, 6)
+
+        ##
+        # Selects lowest criterion score delta
+        scored_objects = [
+            ScoredObject(
+                key=1, score=0.5, variable1=None, variable2=None,
+                rounds=1, wins=None, loses=None, opponents=None
+            ),
+            ScoredObject(
+                key=2, score=0.9, variable1=None, variable2=None,
+                rounds=2, wins=None, loses=None, opponents=None
+            ),
+            ScoredObject(
+                key=3, score=0.5, variable1=None, variable2=None,
+                rounds=2, wins=None, loses=None, opponents=None
+            )
+        ]
+        comparisons = []
+        criterion_scores = {
+            1: { 'c1': 100, 'c2': 100 },
+            2: { 'c1': 110, 'c2': 110 },
+            3: { 'c1': 120, 'c2': 100 }
```

Comment:
can you change c1 to `115` (just to highlight the criteria weighting a bit more and prove that the average isn't being used)

Reply:
updated
https://github.com/ubc/compair/blob/3837f3209f1394e878c852180b7667b9355cb69b/compair/tests/algorithms/test_pair_adaptive_min_delta.py#L216